test/fail/ok/bad-obj-base-types.tc.ok

```diff
@@ -0,0 +1,12 @@
+bad-obj-base-types.mo:8.18-8.20: type error [M0177], ambiguous type field in base
+  t
+bad-obj-base-types.mo:8.11-8.13: info, field also present in base, here (consider overwriting)
```

Comment:
I'd drop the `(consider overwriting)` part for now in the case of type fields.

Reply:
I have a patch for this!

Reply:
Yeah, I agree.

Reply:
Okay to `git push`?

Reply:
Sure, you might need to pull first.

Reply:
Yeah `git pull --rebase` to the rescue! Pushed.